robottelo/helpers.py

```diff
@@ -797,3 +799,78 @@ def from_cmd_str(cls, command='satellite-installer', cmd_str=None):
     def __repr__(self):
         """Custom repr will give the constructed command output"""
         return self.get_command()
+
+
+class settingsUtils:
```

Comment:
I think these methods should largely be part of the `robottelo.hosts.ContentHost`, `Capsule`, or `Satellite` instances, instead of helpers - or just replaced entirely like the case of `capsule_hostname`.

Reply:
@mshriver Ohh ! Misunderstood the Jira ticket or the ticket was very old that didnt have clarity. We should have updated it. Never mind !

I think instead of reworking, I should late @JacobCallahan handle it in his PR of LegacySettings removal. Thoughts @JacobCallahan  ?

Reply:
I'll revisit this on enim past my current phase of the refactor

Reply:
@JacobCallahan Ok. Closing this one in favor of your TODO work. Let me know if you need my help.